### PR TITLE
[WIP]Fix Dask-ML CI

### PR DESCRIPTION
**Pull request by VibhuJawa**
This PR fixes CI 

- [ ] Fix Linting
- [x] Fix Docs Build

**Error Details :** 

This code (see [link](https://github.com/dask/dask-ml/blob/80eb0d81b22df3762706384c3ca1c0be4137e591/dask_ml/datasets.py#L373-L375))   in dask-ml   calls dask.compute  with scheduler="single-threaded"  .  

 This  causes failures in  dask-ml docs  builds due to it raising the below warning  ```UserWarning: Running on a single-machine scheduler when a distributed client is active might lead to unexpected results  and thus is blocking the CI ```
 
**Fix Details:**
Remove the `dask.compute` call  as we dont really need to compute it right away

**Todo:**
- [ ] Address the `test_train_test_split_dask_dataframe_rng` test that popped up

### CODE REVIEW

**Comment by VibhuJawa** [comment]
Closing in favor of https://github.com/dask/dask-ml/pull/929 .
